tests/integration/kafka_test.go

```diff
@@ -327,55 +327,55 @@ func TestKafkaIntegration_bootstrap_inventory(t *testing.T) {
 	}
 }
 
-func TestKafkaIntegration_consumer_offset(t *testing.T) {
-	bootstrapDiscoverConfigInventory := func(command []string) []string {
+func TestKafkaIntegration_producer_test(t *testing.T) {
```

Comment:
no real change, I had just changed the order of the tests to debug it in an easier way